CircularSlider/Classes/CircularSlider.swift

```diff
@@ -48,10 +48,10 @@ open class CircularSlider: UIView {
     fileprivate var backingFractionDigits: NSInteger = 2
     fileprivate let maxFractionDigits: NSInteger = 4
     fileprivate var startAngle: CGFloat {
-        return -CGFloat(M_PI_2) + radiansOffset
+        return -.pi + radiansOffset
```

Comment:
It looks like `M_PI_2` should be replaced with `.pi / 2`. 

**references**
https://stackoverflow.com/questions/43753758/whats-the-difference-between-m-pi-and-m-pi-2/43753803
https://ioscodesnips.wordpress.com/2011/12/20/objective-c-math-functions/